### PR TITLE
Add new Monero Explorer node77.monero.wiz.biz with onion

### PR DESCRIPTION
**Pull request by wiz**
As per https://github.com/bisq-network/proposals/issues/257
cc @chimp1984

### CODE REVIEW

File: core/src/main/java/bisq/core/user/Preferences.java [hunk 1]
```diff
@@ -123,12 +123,14 @@
...
     private static final ArrayList<String> XMR_TX_PROOF_SERVICES_CLEAR_NET = new ArrayList<>(Arrays.asList(
-            "78.47.61.90:8081"));
-    //TODO add a second before release
+            "78.47.61.90:8081", // @emzy
+            "node77.monero.wiz.biz" // @wiz
+    ));
     private static final ArrayList<String> XMR_TX_PROOF_SERVICES = new ArrayList<>(Arrays.asList(
-            "monero3bec7m26vx6si6qo7q7imlaoz45ot5m2b5z2ppgoooo6jx2rqd.onion"));
+            "monero3bec7m26vx6si6qo7q7imlaoz45ot5m2b5z2ppgoooo6jx2rqd.onion", // @emzy
+            "wizxmr4hbdxdszqm5rfyqvceyca5jq62ppvtuznasnk66wvhhvgm3uyd.onion", // @wiz
```
Comment (wiz) [inline core/src/main/java/bisq/core/user/Preferences.java]:
```suggestion
            "wizxmr4hbdxdszqm5rfyqvceyca5jq62ppvtuznasnk66wvhhvgm3uyd.onion" // @wiz
```

File: core/src/main/java/bisq/core/user/Preferences.java [hunk 2]
```diff
@@ -123,12 +123,14 @@
...
-    //TODO add a second before release
     private static final ArrayList<String> XMR_TX_PROOF_SERVICES_CLEAR_NET = new ArrayList<>(Arrays.asList(
-            "78.47.61.90:8081"));
-    //TODO add a second before release
+            "78.47.61.90:8081", // @emzy
+            "node77.monero.wiz.biz" // @wiz
+    ));
     private static final ArrayList<String> XMR_TX_PROOF_SERVICES = new ArrayList<>(Arrays.asList(
-            "monero3bec7m26vx6si6qo7q7imlaoz45ot5m2b5z2ppgoooo6jx2rqd.onion"));
+            "monero3bec7m26vx6si6qo7q7imlaoz45ot5m2b5z2ppgoooo6jx2rqd.onion", // @emzy
```
Comment (sqrrm) [inline core/src/main/java/bisq/core/user/Preferences.java:131]: @Emzy is this correct?

Comment (Emzy) [reply core/src/main/java/bisq/core/user/Preferences.java:131]: Yes, correct.

**Review by sqrrm — APPROVED** [review]

**Comment by sqrrm** [comment]
utACK

I think it's good policy to check with all service providers that their updated onion is correct. Even better if the PR creator could get the affected parties to ack the change.

**Comment by chimp1984** [comment]
@wiz Could you add your clearnet address as well?

**Pull request merged**